loglist2/loglist2.go

```diff
@@ -136,6 +136,59 @@ type TreeHead struct {
 	TreeSize int64 `json:"tree_size"`
 }
 
+// String method returns printable name of the state.
+func (ls *LogStates) String() string {
+	names := [...]string{
+		"Empty",
+		"Pending",
+		"Qualified",
+		"Usable",
+		"Frozen",
+		"Retired",
+		"Rejected"}
+	switch {
+	case ls == nil:
+		return names[0]
```

Comment:
Is there any reason not to just return the values as an immediate, rather than an index into that string array?

Reply:
Done.